.github/workflows/deploy-preview.yml

```diff
@@ -204,16 +227,26 @@ jobs:
         with:
           name: deploy_preview
           status: "${{ job.status }}"
-          url: "${{ steps.deploy.outputs.url }}${{ inputs.relative_prefix }}"
+          url: "${{ steps.deploy.outputs.url }}${{ (fromJSON(inputs.sources))[0].relative_prefix }}"
```

Comment:
I think only a single URL here is fine but it might cause some confusion.

Reply:
Yeah I meant to check if I can put multiple URLs into the status check but I don't think I can